src/sdk/openmldb_api.cc

```diff
@@ -0,0 +1,378 @@
+std::string RequestRow::get_sql() const { return sql; }
+
+RequestRow& RequestRow::operator<<(const bool& value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeBool);
+    record.push_back(value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const int16_t value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeInt16);
+    record.push_back(value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const int32_t value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeInt32);
+    record.push_back(value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const int64_t value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeInt64);
+    record.push_back(value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const TimeStamp value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeTimestamp);
+    record.push_back(value.get_Timestamp());
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const Date value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeDate);
+    record.push_back(value.get_year());
+    record.push_back(value.get_month());
+    record.push_back(value.get_day());
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const float value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeFloat);
+    record.push_back(value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const double value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeDouble);
+    record.push_back(value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const std::string value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeString);
+    str_length += value.length();
+    record.push_back(value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const char* value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeString);
+    str_length += strlen(value);
+    record.push_back((std::string)value);
+    return *this;
+}
+
+RequestRow& RequestRow::operator<<(const OpenmldbNull value) {
+    parameter_types->AddColumnType(::hybridse::sdk::kTypeUnknow);
+    record.push_back(value);
+    return *this;
+}
+
+void RequestRow::reset() {
+    record.clear();
+    parameter_types = std::make_shared<hybridse::sdk::ColumnTypes>();
+    str_length = 0;
+}
+
+bool execute(const OpenmldbHandler& handler, const std::string& sql) {
+    auto rs = (handler.get_router())->ExecuteSQL(sql, handler.get_status());
+    if (rs != NULL) resultset_last = rs;
+    return (handler.get_status())->IsOK();
+}
+
+bool execute(const OpenmldbHandler& handler, const std::string& db, const std::string& sql) {
+    auto rs = (handler.get_router())->ExecuteSQL(db, sql, handler.get_status());
+    if (rs != NULL) resultset_last = rs;
+    return (handler.get_status())->IsOK();
+}
+
+bool execute_parameterized(const OpenmldbHandler& handler, const std::string& db, const std::string& sql,
```

Comment:
why not set the `execute_parameterized` and `execute` as the function members of `OpenmldbHandler`